data/content/environment_variables.yaml

```
@@ -97,17 +97,26 @@ variables:
   example: "cdanvers@kree-net.com"
 - name: BUILDKITE_BUILD_CREATOR
   desc: |
-    The name of the user who created the build. May be **[unverified](#unverified-commits)**.
+    The name of the user who created the build. The value differs depending on how the build was created:
+      Buildkite dashboard: Set based on who manually created the build.
+      GitHub webhook: Set from the  **[unverified](#unverified-commits)** HEAD commit.
+      Webhook: Set based on which user is attached to the API Key used. 
```

Let's keep the same formatting used on the other page:

```suggestion
    The name of the user who created the build. The value differs depending on how the build was created:
      
    - **Buildkite dashboard:** Set based on who manually created the build.
    - **GitHub webhook:** Set from the  **[unverified](#unverified-commits)** HEAD commit.
    - **Webhook:** Set based on which user is attached to the API Key used.
```